server/tests/base_path_test.rs

```diff
@@ -0,0 +1,59 @@
+#[cfg(test)]
+mod base_path_tests {
+    use reqwest::Client;
+    use std::process::{Command, Stdio};
+    use unleash_edge::types::BuildInfo;
+
+    #[actix_web::test]
+    async fn test_base_path() {
+        let base_path = "/nuno/test";
+
+        // Run the app as a separate process
+        let mut app_process = Command::new("./../target/debug/unleash-edge")
+            .arg("-b")
+            .arg(&base_path)
+            .arg("edge")
+            .arg("-u")
+            .arg("http://localhost:4242")
+            .stdout(Stdio::null()) // Suppress stdout
+            .stderr(Stdio::null()) // Suppress stderr
+            .spawn()
+            .expect("Failed to start the app");
+
+        // Wait for the app to start up
+        std::thread::sleep(std::time::Duration::from_secs(2));
+
+        // Send a request to the app
+        let client = Client::new();
+        let base_url = "http://localhost:3063";
+        let endpoint = "/internal-backstage/info";
```

Comment:
I think that's fine! If @sighphyre or @chriswk have a better suggestion for this test we can definitely do this afterwards

Reply:
>Well, I wonder if we have something we can test that also returns 200, but in any case, a 403 might indicate that the endpoint is properly configured (otherwise it'd be a 404). Although I would prefer to have something that returns 200, I'm not sure if that's possible...

This was a great catch @gastonfournier, this looked totally fine to me at a glance but I went and checked - this doesn't work on Edge, if we don't get a token, we assume you're trying to do something naughty and you get a blanket 403 back so non configured endpoints will still return a 403

What we can do is spin up edge in offline mode and provide it a token - doesn't need to be a valid token, offline mode will accept anything that _looks_ like a Unleash token. We should be able to use the bootstrap file in `examples/features.json` for that

Reply:
Something like this instead? https://github.com/Unleash/unleash-edge/pull/191/commits/6982f45cfee5601c620d2b229b0ed396deee6ca9

Reply:
Love it!